distributed/nanny.py

```diff
@@ -506,18 +509,15 @@ async def close(self, comm=None, timeout=5, report=None):
         for preload in self.preloads:
             await preload.teardown()
 
-        self.stop()
```

Comment:
Why this change?

Reply:
It is redundant since we do the same in `Server.close`. it looked like being redundant and I felt removing it would reduce complexity. If there is some purpose to stopping earlier, I can revert this logic. This is only my sense of _cleaning up_